src/GameServer/Systems/Player/TeamManager.cs

```diff
@@ -160,7 +163,7 @@ public int GetMaxTeamSize()
                 return GameServer.Configuration.Server.GameOptions.AvatarLimits.SinglePlayerTeam;
         }
 
-        public async Task UpdateTeamEntities()
+        public async Task UpdateTeamEntitiesAsync(BasePacket packet=null)
```

Comment:
This is my only complaint. As I have stated before, The corresponding rsp packets should be sent only by the matching req handlers
```suggestion
        public async Task UpdateTeamEntitiesAsync()
```